patches/minecraft/net/minecraft/resources/RegistryResourceAccess.java.patch

```diff
@@ -30,8 +38,8 @@
           }
  
           private static String m_214239_(ResourceLocation p_214240_) {
-+            if (net.minecraftforge.registries.RegistryManager.FROZEN.getRegistry(p_214240_) != null && !(p_214240_.m_135827_().equals("minecraft")))
-+               return p_214240_.m_135827_() + "/" + p_214240_.m_135815_(); // FORGE: add non-vanilla registry namespace to loader directory, same format as tag directory (see net.minecraft.tags.TagManager#getTagDir(ResourceKey))
-             return p_214240_.m_135815_();
+-            return p_214240_.m_135815_();
++            return DataPackRegistriesHooks.prefixNamespace(p_214240_);// FORGE: add non-vanilla registry namespace to loader directory, same format as tag directory (see net.minecraft.tags.TagManager#getTagDir(ResourceKey))
```

Comment:
Try to have a space between the line termination and comments please. Same for the other patch file.